### PR TITLE
Issue 100 ft.info

### PR DESCRIPTION
**Pull request by tgrall**
I have updated the library to Jedis 3.4.1 and use the new encode method

Note: I was not able to run all the tests using `mvn test` (even on master branch) but all test are passing from the ide

### CODE REVIEW

**Comment by tgrall** [comment]
@gkorland can please review

File: src/test/java/io/redisearch/client/ClientTest.java [hunk 1]
```diff
@@ -11,11 +11,7 @@
 import redis.clients.jedis.exceptions.JedisDataException;
 import redis.clients.jedis.util.SafeEncoder;
 
-import java.util.Arrays;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```
Comment (sazzad16) [inline src/test/java/io/redisearch/client/ClientTest.java:14]:
```suggestion
import java.util.Arrays;
import java.util.HashMap;
import java.util.List;
import java.util.Map;
import java.util.Set;
```

File: src/test/java/io/redisearch/client/ClientTest.java [hunk 2]
```diff
@@ -522,8 +518,8 @@ public void testAlterAdd() throws Exception {
         
         Map<String, Object> info = cl.getInfo();
         assertEquals(TEST_INDEX, info.get("index_name"));
-        assertEquals("tags", SafeEncoder.encode(((List<List<byte[]>>)info.get("fields")).get(1).get(0)));
-        assertEquals("TAG", SafeEncoder.encode(((List<List<byte[]>>)info.get("fields")).get(1).get(2)));
+        assertEquals("tags",((List)((List)info.get("fields")).get(1)).get(0) );
+        assertEquals("TAG", ((List)((List)info.get("fields")).get(1)).get(2) );
```
Comment (sazzad16) [inline src/test/java/io/redisearch/client/ClientTest.java:522]:
```suggestion
        assertEquals("tags",((List)((List)info.get("fields")).get(1)).get(0));
        assertEquals("TAG", ((List)((List)info.get("fields")).get(1)).get(2));
```

File: src/test/java/io/redisearch/client/ClientTest.java [hunk 3]
```diff
@@ -581,12 +577,32 @@ public void testPhoneticMatch() throws Exception {
...
+
         assertTrue(cl.createIndex(sc, Client.IndexOptions.defaultOptions()));
 
         Map<String, Object> info = cl.getInfo();
         assertEquals(TEST_INDEX, info.get("index_name"));
 
+        assertEquals(6, ((ArrayList)info.get("fields")).size());
+        assertEquals("global_idle", ((ArrayList)info.get("cursor_stats")).get(0));
+        assertEquals(0L, ((ArrayList)info.get("cursor_stats")).get(1));
+
```
Comment (sazzad16) [inline src/test/java/io/redisearch/client/ClientTest.java:605]:
```suggestion
        assertEquals(6, ((List)info.get("fields")).size());
        assertEquals("global_idle", ((List)info.get("cursor_stats")).get(0));
        assertEquals(0L, ((List)info.get("cursor_stats")).get(1));
```

**Review by sazzad16 — APPROVED** [review]

**Pull request merged**